ui/analyse/src/ctrl.ts

```diff
@@ -947,4 +965,24 @@ export default class AnalyseCtrl {
 
   withCg = <A>(f: (cg: ChessgroundApi) => A): A | undefined =>
     this.chessground && this.cgVersion.js === this.cgVersion.dom ? f(this.chessground) : undefined;
+
+  userJumpPlyDelta = (plyDelta: number) => {
+    if (plyDelta === -1) control.prev(this);
+    else if (plyDelta === 1) control.next(this);
+    else if (plyDelta > 1) control.last(this);
+    else if (plyDelta < -1) control.first(this);
+  };
+
+  auxMove = (orig: cg.Key, dest: cg.Key, prom: cg.Role | undefined) => {
+    const capture = this.chessground.state.pieces.get(dest);
+    this.sendMove(orig, dest, capture, prom);
+  };
+
+  auxUpdate = (fen: string) => {
```

Comment:
I believe the intention was "auxiliary update", but agreed even spelled out it could be improved. Its purpose is to fan out updates to the various plugins, and I think it existing at all will make more sense when we implement voiceMove. We could name it something like `updatePlugins`?

Reply:
pluginUpdate and pluginMove sound good.

Reply:
In any case, made that naming change here: https://github.com/lichess-org/lila/pull/15019/commits/10b0d242fbdc789a9679f0db86ea091f8858b258

Reply:
As for `pluginMove`, something like `handlePluginMove` makes the most sense to me. If I'm in a controller and see that function, I know that we are handling a move that's coming from the plugin. I'm always reminding myself which way the data is flowing, and in theory this could help with that.

Reply:
the actor followed by the verb is a convention shared by rounds, puzzles, and chessground. i.e userMove, apiMove, etc.

but i don't care at all, i'm the one who chose `auxMove` so just shut me in a soundproof closet and pick something - it's fine.

Reply:
That's fair, consistency is important. Too light of a preference to break with consistency or shove anyone in a closet I think. Side note, I've loved seeing the refactored controllers (functions->classes) and overall the code is in a great spot it feels like, a nice departure from my work's codebase 😌